python/SeanZvvHbb13TeVSteps/README.md

```diff
@@ -0,0 +1,71 @@
+##Instructions for ZvvHbb on LXPLUS
+
+*Disclaimer: This package is under active development. The default branch, V20_from_silvio, is guaranteed to run at PSI or Pisa. The version that David uses runs on LXPLUS, but may or may not be synchronized with the default branch. The default branch is being tested on LXPLUS and, pending potential modifications and updates, should be verified to run on LXPLUS by the end of next week.*
```

Comment:
Ciao @swang373 just to be clear: I fully appreciate what you are doing, and I regret not having been able to ensure better communication.
After your first effort to use xrootd, I profited to implement a similar thing building on existing blocks and structure.
I setup the so called copytreePSI.py
https://github.com/perrozzi/Xbb/blob/V20_from_silvio/python/myutils/copytreePSI.py
which has more features integrated with submitThem.py to speed up the prep step (for TT_powheg it can take days without splitting the dataset in several jobs)
I believe it would have been (and still is) easy to adapt to the lxplus needs
(bsub instead of qsub, maybe some light change of protocol, but in any case both PSI and CERN are T2 so most of the features are very similar).
I committed and pushed to your branch to let you see, probably I should have stressed more this and commented more in my code.
I am available for clarifying chats anytime